### ISSUE TITLE
More work needed in overlay_microtxn_authdialog.layout

### ISSUE THREAD

**Issue by badanka** [badanka/Compact#31]
I have adjusted this as far as I'm able. I have no idea what it looks like if you get your price converted to another currency though. I'm looking for people who can help me with this.

![microtransaction_overlay_cr](https://user-images.githubusercontent.com/627377/28122340-37d05890-671e-11e7-8048-fe968a41ea4d.png)

**Comment by avsaponchik** [comment]
Here's mine
![default](https://user-images.githubusercontent.com/1337953/28128113-55a1dd26-6737-11e7-860c-513160b1fc84.png)

**Comment by badanka** [comment]
Thanks, at least that tells me that it seems to look decent in Russian/Belarusian as well. There were some new additions in the form of updating billing information and recurring payments. Do you happen to know of any games that have recurring payments through the Steam overlay system? I'm not very well versed in games containing microtransactions and monthly payments.

**Comment by avsaponchik** [comment]
@badanka nah... i only play few games like cs, saints row, gtaiv and such

**Comment by iTz-i-EzzA** [comment]
Hey man good work i love it. is there a way i can change the colour blue to green?. thanks

**Comment by badanka** [comment]
Change blue to green where? What part is it you want to change? And in what way is this related to the overlay transaction window?

**Comment by danielbugs** [comment]
If I'm not to late here it is for the Euro € currency.
![price_test](https://user-images.githubusercontent.com/15776544/30876170-b72a1a6c-a2ed-11e7-9f28-4898720ac1b6.png)
